app/models/concerns/imageable.rb

```diff
@@ -0,0 +1,23 @@
+module Imageable
+  extend ActiveSupport::Concern
+
+  included do
+    mount_uploader :image, ImageUploader
+
+    after_commit :attach_image_from_discogs, on: :create
```

Comment:
This callback might need to remove, because when user syncing their library,  this will cause a lots of request to Discogs API. Like the another PR you have mentioned, this may cause a rate limit issue on the Discogs API.

Reply:
Ah yes, I will remove this until we address the rate limit issue.